website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -432,12 +432,6 @@ around to distributed workers). The Beam SDKs provide a data encoding mechanism
 that includes built-in encoding for commonly-used types as well as support for
 specifying custom encodings as needed.
 
-{{< paragraph class="language-go" >}}
```

Comment:
Does the generics approach also remove the need to register structs?

Reply:
Yeah, all the structs will be inferred and registered automatically